src/OpenIddict.Quartz/OpenIddictQuartzConfiguration.cs

```diff
@@ -34,6 +34,7 @@ public void Configure(QuartzOptions options)
             // reached if the application is shut down or recycled. As such, this trigger is set up to fire
             // between 1 and 10 minutes after the application starts to ensure the job is executed at least once.
             builder.ForJob(OpenIddictQuartzJob.Identity)
+                   .WithIdentity(OpenIddictQuartzJob.Identity.Name, OpenIddictQuartzJob.Identity.Group)
```

Comment:
@lahma hey Marko 👏🏻 

I have a question for you, if you don't mind: instead of using "programmatic names" for the names of the jobs/triggers, would it make sense to use better-looking names so they are more... "human-readable"? Something like:
  - `JobKey.Name`: OpenIddict job
  - `JobKey.Group`: OpenIddict/Quartz.NET integration
  - `TriggerKey.Name`: OpenIddict built-in trigger
  - `TriggerKey.Group`: OpenIddict/Quartz.NET integration

https://www.quartz-scheduler.net/documentation/best-practices.html#job-tips doesn't have any specific recommendation regarding names, but I'm wondering if it wouldn't be a good idea for users that configure their Quartz job via an UI like Quartzmin, where the job/trigger key/group will be displayed. Is it something you'd recommend/encourage?

Thanks!

Reply:
Well I don't have and solid advice, but things to consider:

* group is very useful concept that you can also to do batch operations like pause all and resume all in group, so sharing a group name like you have in you configuration example probably is the most useful thing, the actual contents don't matter that much - there's also "starts with" kind of queries but probably don't apply here
* triggers and jobs also have `Description` which might also be usable if shown somewhere

Reply:
Great, thanks a lot for the insights!

We already assigned the job a group and a description, but the trigger itself didn't have a static `TriggerKey` :+1: